e2e-tests/cypress/integration/appellant-submission-check-your-answers.feature

```diff
@@ -0,0 +1,13 @@
+@wip
+Feature: A user checks their answers and wants to submit their appeal
```

Comment:
I think simply by convention we have been using Prospective appellant in the Feature and Scenario titles and User in the steps.